releasenotes/README.md

```diff
@@ -1,259 +1,105 @@
-
-## 49.3.0 (July 23, 2020)
-
-* We aren't releasing a Salesforce CLI patch this week.
-
-## 49.2.3 (July 18, 2020) 
-
-Welcome to the first release of version 49 of the `salesforcedx` CLI plug-in in Summer '20. 
+[Click here for the v49 release notes.](./v49.md)
 
 ### Installation Notes
 
-Update Salesforce CLI to v49 by running `sfdx update` or any CLI command.
+Update Salesforce CLI to v50 by running `sfdx update`.
 
 ```bash
 $ sfdx update
 ```
 
 If you are installing Salesforce CLI for the first time, see [Install Salesforce CLI](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm#sfdx_setup_install_cli) to install the CLI for your operating system, then run `sfdx update`.
 
-Run `sfdx version` to display the version of Salesforce CLI installed on your computer. Run `sfdx plugins --core` to display the version of the installed `salesforcedx` plug-in.
+Run `sfdx plugins --core` to display the version of Salesforce CLI and the installed `salesforcedx` plug-in.
 
 ```bash
 $ sfdx plugins --core
+
 @oclif/plugin-autocomplete 0.1.5 (core)
 @oclif/plugin-commands 1.3.0 (core)
-@oclif/plugin-help 3.0.1 (core)
+@oclif/plugin-help 3.2.0 (core)
 @oclif/plugin-not-found 1.2.4 (core)
-@oclif/plugin-plugins 1.7.10 (core)
+@oclif/plugin-plugins 1.9.0 (core)
 @oclif/plugin-update 1.3.10 (core)
 @oclif/plugin-warn-if-update-available 1.7.0 (core)
 @oclif/plugin-which 1.0.3 (core)
-@salesforce/sfdx-trust 3.0.7 (core)
-analytics 1.12.0 (core)
+@salesforce/sfdx-trust 3.4.3 (core)
+alias 1.1.2 (core)
+config 1.1.8 (core)
 generator 1.1.3 (core)
-salesforcedx 49.2.3                  // salesforcedx plug-in version
-├─ salesforcedx-templates 48.32.0
-├─ custom-metadata 1.0.7
-├─ salesforce-alm 49.3.0
-└─ @salesforce/sfdx-plugin-lwc-test 0.1.7
-sfdx-cli 7.66.2 (core)               // CLI version
+salesforcedx 50.1.0 (core)                    // salesforcedx plug-in version
+├─ templates 50.1.0 (core)
+├─ custom-metadata 1.0.10 (core)
+├─ @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
+├─ apex 0.1.0 (core)
+└─ salesforce-alm 50.1.0 (core)
+sfdx-cli 7.75.1 (core)                        // Salesforce CLI version
 ```
 
+If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_dev_cli_autocomplete.htm), run `sfdx autocomplete --refresh-cache` after you update the `salesforcedx` plug-in to ensure that autocomplete works correctly on any new commands.
+
 ### Uninstall the Pre-Release Version of the salesforcedx CLI Plug-In
 
-If you installed the pre-release version of the `salesforcedx` v49 plug-in to test the next version of Salesforce CLI, uninstall it and update the CLI. We’re no longer updating the pre-release v49 plug-in.
+If you installed the pre-release version of the `salesforcedx` v50 plug-in, uninstall it and update the CLI. We’re no longer updating the pre-release v50 plug-in.
 
-```bash
+```bassh
```

Comment:
extra "s" in bassssssh???

Reply:
Interestingly, when I look at the preview version of the rel notes, I don't see this.

Reply:
When I look at the preview, I don't see this.

Reply:
Ha, funny, I think that bassssssssh tag/whatever is ignored because I think it's all rendering correctly.  Right?  In any case, I'll fix it.

Reply:
And to be clear, you're not supposed to see the word "bash" in the rendering, it's just for identifying a bash code block. I think it's superfluous here, so I'll probs stop using it.